version.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+var (
+	// BuildDate is date when binary was built
+	BuildDate string
+	// BuildVersion is the version of binary
+	BuildVersion string
+)
+
+// providerVersion holds current provider version
+type providerVersion struct {
```

Comment:
Can this instead be a type exported from the CSI driver? Seems a little odd for each provider to create it, while still needing to conform to a standard format.

Reply:
@kalafut I don't think we want to add driver as a dependency here for the version type. WDYT?

Reply:
Yeah I've had a change of heart as this is such a simple structure. Let's leave as is.